src/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj

```diff
@@ -58,7 +58,16 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <None Include="Data\CodePageNameMappings.csv" />
+    <None Include="Data\PreferredCodePageNames.csv" />
     <None Include="project.json" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+  <!-- Generator for code mapping table, target to invoke is GenerateEncodingSource -->
```

Comment:
@tarekgh - Done, new PR is [239](https://github.com/dotnet/buildtools/pull/239).  Changed a few things here to enable printing of source file names.